packages/compiler/core/types.ts

```diff
@@ -1276,6 +1289,46 @@ export enum IdentifierKind {
   Other,
 }
 
+// Doc-comment related syntax
```

Comment:
use a `# region` for this?

Reply:
Not opposed, but we have other comments like this (e.g. for projection nodes). Maybe best to do separately across all of them?

Reply:
Filed #1316